src/main/scala/com/github/unchama/seichiassist/data/HalloweenItemData.scala

```diff
@@ -41,37 +68,74 @@ object HalloweenItemData {
     nbtItem.getItem
   }
 
-  private val halloweenPotionItemFlags = Set(
-    ItemFlag.HIDE_ENCHANTS,
-    ItemFlag.HIDE_POTION_EFFECTS
-  )
-
-  private val halloweenPotionEffects = Set(
-    new PotionEffect(PotionEffectType.REGENERATION, 20 * 10, 3),
-    new PotionEffect(PotionEffectType.DAMAGE_RESISTANCE, 20 * 20, 2),
-    new PotionEffect(PotionEffectType.NIGHT_VISION, 20 * 60 * 10, 0),
-    new PotionEffect(PotionEffectType.LUCK, 20 * 60 * 5, 0)
-  )
-
-  private def halloweenPotionLoreList() = {
-    val year = Calendar.getInstance().get(Calendar.YEAR)
-    List(
-      "",
-      s"${ChatColor.RESET}${ChatColor.GRAY}${year}ハロウィンイベント限定品",
-      s"${ChatColor.RESET}${ChatColor.GRAY}敵に囲まれてピンチの時や",
-      s"${ChatColor.RESET}${ChatColor.GRAY}MEBIUS育成中の時などにご利用ください"
+  def isHalloweenPotion(itemStack: ItemStack): Boolean = {
+    if (itemStack != null && itemStack.getType != Material.AIR) {
+      new NBTItem(itemStack).getByte(NBTTagConstants.typeIdTag) == 1
+    } else {
+      false
+    }
```

Comment:
```suggestion
    itemStack != null && itemStack.getType != Material.AIR && {
      new NBTItem(itemStack)
        .getByte(NBTTagConstants.typeIdTag) == 1
    }
```
ブロックを使えばifより単純になります( `isHalloweenHoe` もこうしてよさそう)